XML/Snippet.sublime-syntax

```diff
@@ -0,0 +1,26 @@
+%YAML 1.2
+---
+# See http://www.sublimetext.com/docs/3/syntax.html
+file_extensions:
+  - sublime-snippet
+name: Sublime Text Snippet
+scope: text.xml.sublime-snippet
+contexts:
+  main:
+    - match: '(<)(content)(>)(<!\[CDATA\[)'
```

Comment:
The `<![CDATA[` part may be provided on the next line (or even be skipped entirely) afaik.